examples/motor/m3508_steering.cc

```diff
@@ -51,63 +54,89 @@ bool steering_align_detect() {
 
 void RM_RTOS_Init() {
   print_use_uart(&huart8);
-  bsp::SetHighresClockTimer(&htim2);
+  bsp::SetHighresClockTimer(&htim5);
+
+  can1 = new bsp::CAN(&hcan1, 0x201, true);
+  motor1 = new control::Motor3508(can1, 0x201);
+  motor3 = new control::Motor3508(can1, 0x203);
 
-  can1 = new bsp::CAN(&hcan1, 0x201);
-  motor = new control::Motor3508(can1, 0x202);
+  key = new bsp::GPIO(KEY_GPIO_GROUP, KEY_GPIO_PIN);
```

Comment:
Add a comment here - what is the purpose of key? I am assuming that key on the dev board is pressed to trigger the alignment process

Reply:
Resolved in 3dea93c and b145258